roles/jd-server/README.md

```diff
@@ -0,0 +1,27 @@
+# JD Server
+
+The JD Server is a Sv2 proxy, It is also the entity responsible for Pool-side block propagation in case of valid blocks found by miners connected to the pool (who are using the Job Declaration Protocol).
+
+## Setup
+
+### Configuration File
+
+The configuration file contains the following information:
+
+1. The SRI Pool information which includes the SRI Pool authority public key (`authority_public_key`), the SRI Pool authority secret key (`authority_secret_key`), along with its certificate validity (`cert_validity_sec`). In addition to this, it contains the address which it will use to listen to new connection from downstream roles (`listen_jd_address`) and the list of uncompressed pubkeys for coinbase payout (`coinbase_outputs`).
```

Comment:
```suggestion
1. The SRI Pool information which includes the SRI Pool authority public key (`authority_public_key`), the SRI Pool authority secret key (`authority_secret_key`). 
2. The list of uncompressed pubkeys for coinbase payout (`coinbase_outputs`)
3. The address which it will use to listen to new connection from downstream roles (`listen_jd_address`)
```